### PR TITLE
Export logger extension so it can be used by client

### PR DESCRIPTION
**Pull request by bdgamble**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,56 +1,4 @@
...
-						.resolve(utils.callErrorHook(options.onError, err, event, newContext))
-						.then(result => utils.callHook(options.onAfter, result, event, newContext))
-						.then(result => callback(null, result));
-				})
-			)
-			.catch(err => callback(utils.sanitizeError(err, options)));
-	};
-};
+module.exports = require('./lib/handler');
+module.exports.loggerExtension = require('./lib/logger-extension');
```
Comment (AntonBazhal) [inline index.js]: What do you think about `module.exports.extensions.logger`? Not sure about it, though.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "lambda-handler-as-promised",
-  "version": "1.2.0",
+  "version": "1.3.0",
```
Comment (AntonBazhal) [inline package.json]: It's not a breaking change, so maybe `1.2.1` is enough?

**Comment by bdgamble** [comment]
not sure why circleci isn't responding, I can't see the build in there.

**Comment by AntonBazhal** [comment]
Because building PRs from forks was not activated. Can you reopen?

**Comment by bdgamble** [comment]
@AntonBazhal this ok to merge?

**Pull request merged**